### PR TITLE
fix Contact us en_US and mail address

### LINKED ISSUE #1788: About Us - The label for 'contact us email' need to be in sync for all languages.
1. Login into 'staging.fightpandemics.work' and navigate to 'About Us' page.
2. Scroll down on 'about Us' page to view the 'contact email'

Issue: You will observe that for en_US i.e. American English, the label for 'contact Us' email is 'Contact us at:' but for all the other languages it is 'Or contact us at:'
To keep the label in sync we need to update 'en_US language context. Also, currently in PROD we have 'Or contact us at:' it is so because we have given user the option to either contact us through '#com..' or contact us through email.

Refer below screenshot.
image

### PR DESCRIPTION
**Pull request by ddveloper**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_
to fix #1788 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/locales/translations/en_US.json [hunk 1]
```diff
@@ -369,7 +369,7 @@
     "ourIdea": "Our Idea",
     "ourSupporters": "Our Supporters",
     "followUs": "Follow us on social media",
-    "contactUs": "Contact us at: ",
+    "contactUs": "Or contact us at: ",
     "contactEmail": "contact@fightpandemics.com",
```
Comment (joshmorel) [inline client/src/locales/translations/en_US.json:373]: Your fix to the reported issue is good - thanks @ddveloper  ! However, I'm wondering why do we `contactEmail` as a translation item?

**Review by joshmorel — APPROVED** [review]

File: client/src/locales/translations/en_US.json (hunk 1 above)
Comment (ddveloper) [reply client/src/locales/translations/en_US.json:373]: you're right @joshmorel , the only place uses this is in the AboutUs page showing the exact same characters, I guess we can delete that pair, or put it somewhere under the `constants` folder

**Review by rabbi08 — APPROVED** [review]
Verified the fix on feature branch and it is working fine. Refer below screenshot. Thanks
image

@ddveloper @joshmorel we can update in translation to avoid issues in future

**Comment by rabbi08** [comment]
Verified the fix in PROD. Thanks @ddveloper @joshmorel 

Refer below GIF:
ezgif com-gif-maker (51)

**Pull request merged**